modules/visualization/dev-instance/stack.py

```diff
@@ -142,7 +142,7 @@ def __init__(
                     exclude_punctuation=True,
                     include_space=False,
                     exclude_characters="',. |<>=/\"\\\$#;@[]{}~:`",
-                    password_length=6,
+                    password_length=24,
```

Comment:
So the user has to add the pwd on the NiceDCV login and on the ubuntu login, correct?  For NiceDCV, they can cut/paste the output from SecretsManager, but for the ubuntu login, they have to type in the entire pwd (case-sensitive).  Is ti a good idea to have them punch in a 24 char case-sensitive pwd?  I would reconsider this....

Reply:
The time required to brute force crack a 6 character password with GPU advances has dropped so significantly, it can't be assumed secure at all. For instance, see the reference table https://www.techrepublic.com/article/how-an-8-character-password-could-be-cracked-in-less-than-an-hour/

Recommendation is to leave this at 24 characters as default and allow specifying a lower length via parameters

Reply:
Agreed!